src/app/public/src/modules/clipboard/clipboard.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, ElementRef, AfterViewInit, Renderer2 } from '@angular/core';
+
+@Component({
+  selector: 'stache-clipboard',
+  templateUrl: './clipboard.component.html',
+  styleUrls: ['./clipboard.component.scss']
+})
+export class StacheClipboardComponent implements AfterViewInit {
+  private defaultTigger = 'clipboardButton';
+  private clipboardButton: HTMLButtonElement;
+
+  @Input()
+  public customClickHandler: boolean = false;
+
+  @Input()
+  public buttonText: string = 'Copy to Clipboard';
```

Comment:
These English strings should be referenced from `SkyAppResourcesService`.
https://developer.blackbaud.com/skyux/learn/get-started/advanced/localization